prepare_stmt.go

```diff
@@ -63,7 +67,6 @@ func (sdb *PreparedStmtDB) Reset() {
 	for _, stmt := range sdb.Stmts {
 		go stmt.Close()
 	}
```

Comment:
`Reset` will have similar problems as `Close`. Can we changed it?

Reply:
> `Reset` will have similar problems as `Close`. Can we changed it?

OK, will do it.